Mathlib/Combinatorics/SimpleGraph/Turan.lean

```diff
@@ -280,6 +270,26 @@ theorem nonempty_iso_turanGraph : Nonempty (G ≃g turanGraph (Fintype.card V) r
 
 end IsTuranMaximal
 
-end Forward
+variable (hr : 0 < r)
+
+/-- **Turán's theorem**, reverse direction.
+
+Any graph isomorphic to `turanGraph n r` is itself Turán-maximal. -/
+theorem isTuranMaximal_of_iso (f : G ≃g turanGraph n r) : G.IsTuranMaximal r := by
```

Comment:
Could you please generalize this to say that if `G1 ≃g G2` then `G1.IsTuranMaximal r` iff `G2.IsTuranMaximal r`?

Most of this proof would then turn into a proof of `isTuranMaximal_turanGraph`.

Reply:
I've added the generalisation as `IsTuranMaximal.iso` – it is easier and more readable to prove the iff statement from the original theorem than to prove the iff statement directly.